api/lagoon/client/_lgraphql/organizations/addOrganization.graphql

```diff
@@ -0,0 +1,9 @@
+mutation (
+  $name: String!,) {
+  addOrganization(input: {
```

Comment:
addOrganization supports additional optional fields on creation, these should be exposed in the mutation too so that I can create an organization up front with a specific quota or other option if required